### ISSUE TITLE
Research captioning/transcription vendors

### ISSUE THREAD

**Issue by escowles** [pulibrary/figgy#1878]
* UXO can recommend some captioning vendors
* Maybe digitization vendors can do captioning?
* Cornell is doing a pilot on in-house captioning with student workers

**Comment by escowles** [comment]
Don Skemer and Kelly Bolding recommended looking at temi.com (automated but apparently very good) — though they are English-only.

**Comment by hackmastera** [comment]
General searching, ask on mailing lists, slack channels.
We've had trouble getting the list from UXO.

**Comment by tpendragon** [comment]
@kelynch There's a working group doing this right? Maybe close this?

**Comment by tpendragon** [comment]
There's a working group.
